### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by Franekskc**
- Fix error messages not showing in gui
- Delete prints in console
- Fix detecting face before detecting reference
- Change file validation process

### CODE REVIEW

File: implementation/GUI/gui.py [hunk 1]
```diff
@@ -119,8 +138,8 @@ def not_ready_to_measure(self):
                 or len(self.scene.left_eye_points) != 2
                 or len(self.scene.right_eye_points) != 2
                 or len(self.scene.reference_points) != 4
-                or int(self.le_referenceMM.text()) < 1
```
Comment (stttkm) [inline implementation/GUI/gui.py:122]: now our app accepts negative value for the length of a reference

File: implementation/GUI/gui.py [hunk 2]
```diff
@@ -21,6 +22,10 @@ def __init__(self, backend):
         # things in the graphics area
         self.image = None  # qtg.QPixmap
 
+        # two types of images
+        self.np_image = None
```
Comment (stttkm) [inline implementation/GUI/gui.py]: idk if np_image and mp_image should be in GUI when they are clearly a backend responsibility.

**Review by stttkm — CHANGES_REQUESTED** [review]
- negative reference length is now accepted
- np_image and mp_image belong to the backend, not to the class GUI

File: implementation/GUI/gui.py (hunk 2 above)
Comment (Franekskc) [reply implementation/GUI/gui.py]: If image are not stored in class, every time we try to detect reference/face, it needs to load image from the beggining. I agree that it's not a gui responsibility so I will move these attributes to Backend class

File: implementation/GUI/gui.py (hunk 1 above)
Comment (Franekskc) [reply implementation/GUI/gui.py:122]: Sorry, I have removed the check for a positive value, but added a check for any value given. Now both conditions are checked

**Pull request merged**